### PR TITLE
Add instructions for setting up a new chain.

### PR DESCRIPTION
**Pull request by measure-fi**
https://github.com/palomachain/paloma/issues/139

# Background

We need instructions for setting up a new chain, both from scratch and connecting to an existing one.

# Testing completed

- [x] Set up a testnet with these instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,96 @@
...
 
+## Setting up a new chain from scratch.
+
+### Initializing a new chain
+
+Download the latest release of paloma.
+
+```bash
+wget -O - https://github.com/palomachain/paloma/releases/download/v0.0.1-alphawasmd3/paloma_0.0.1-alphawasmd3_Linux_x86_64.tar.gz | \
+tar -C /var/local -xvzf - paloma
```
Comment (Vizualni) [inline README.md:11]: the binary is actually palomad (I made the initial mistake in the first release)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,96 @@
...
+git clone https://github.com/palomachain/paloma
+PALOMA_CMD=/var/local/paloma ./paloma/scripts/setup-chain-validator.sh
+```
+
+### Starting the paloma daemon
+
+We should now see an error free execution steadily increasing chain depth
+
+```bash
+/var/local/paloma start
```
Comment (Vizualni) [inline README.md:34]: palomad

File: README.md [hunk 3]
```diff
@@ -1,4 +1,96 @@
...
+```
+
+Stake your funds and start `palomad`.
+```bash
+MAIN_VALIDATOR_NODE="tcp://157.245.76.119:26657"
+CHAIN_ID="paloma"
+DEFAULT_GAS_AMOUNT="10000000"
+VALIDATOR_STAKE_AMOUNT=100000dove
+PUBKEY="$(/var/local/paloma tendermint show-validator)"
+/var/local/paloma tx staking create-validator \
```
Comment (Vizualni) [inline README.md:72]: palomad

File: README.md [hunk 4]
```diff
@@ -1,4 +1,96 @@
 # Paloma
 
+## Setting up a new chain from scratch.
```
Comment (Vizualni) [inline README.md:3]: could you add this after those badges?

File: README.md (hunk 1 above)
Comment (measure-fi) [reply README.md:11]: Yep. And I can change this to use "latest".

File: README.md (hunk 4 above)
Comment (measure-fi) [reply README.md:3]: Ah that's what that is. Definitely. I also need to rebase on @taariq's changes after https://github.com/palomachain/paloma/pull/133 and https://github.com/palomachain/paloma/pull/134 are merged.

**Review by rootedbox — COMMENTED** [review]
👍

**Review by rootedbox — APPROVED** [review]

**Pull request merged**